### PR TITLE
Button group

### PR DESCRIPTION
**Pull request by bartderijk**
I added the button group component, it can take some buttons and will group them together just like it does in the funda ui button group component. Nothing fancy (yet), it just does exactly the same job as it's predecessor. It's very minimal and basic, a good start for iteration I figure. Let me know what you think.

Schermafbeelding 2020-11-26 om 13 48 03

### CODE REVIEW

File: src/components/ui-button-group.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        class="fd-btn-group inline-flex"
+        role="group"
+        aria-label="Button group"
+    >
+        <slot />
+    </div>
+</template>
+
+<style scoped>
+.fd-btn-group button:not(:last-child),
```
Comment (bartderijk) [inline src/components/ui-button-group.vue:12]: I attempted to have these styles applied using tailwind, but no sigar. I have not yet found a way to do it using their utility classes.

Comment (dine) [reply src/components/ui-button-group.vue:12]: I really don't like this styles here, specially the `:not` selector. 
Maybe something using only :first-child and :last-child makes it easier to read?

Comment (bartderijk) [reply src/components/ui-button-group.vue:12]: I agree, tho the styles are an exact copy of what we're already using on funda ui.

Comment (bartderijk) [reply src/components/ui-button-group.vue]: @dine I refactered it so it's a bit cleaner.

**Review by dine — APPROVED** [review]

**Pull request merged**